Games/types/Mafia/roles/cards/PotionCaster.js

```diff
@@ -0,0 +1,160 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_DEFAULT, PRIORITY_NIGHT_SAVER, PRIORITY_INVESTIGATIVE_DEFAULT } = require("../../const/Priority");
+
+module.exports = class PotionCaster extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Cast Potion": {
+                states: ["Night"],
+                flags: ["voting"],
+                action: {
+                    priority: PRIORITY_NIGHT_SAVER - 1,
+                    run: function () {
+                        // set target
+                        this.actor.role.data.currentTarget = this.target;
+                    }
+                }
+            },
+            "Choose Potion": {
+                states: ["Night"],
+                flags: ["voting"],
+                inputType: "custom",

```

Comment:
@R9H9 it is for this part, I would have to set inputType to "alignment" to work but "alignment" is not exactly right here so I used custom